src/navigators/Navigator.tsx

```diff
@@ -40,6 +40,7 @@ interface MapStateToProps {
   isAuthenticated: boolean;
   isTxPasswordSet: boolean;
   isNotificationEmailSet: boolean;
+  isNotificationEmailSkip: boolean;
```

Comment:
is there a case when `isNotificationEmailSkip` and `isNotificationEmailSet` are different? If no it would be better to keep it as one flag with generic name

Reply:
yes in your scenario and logic when u will delete email :)